app/models/concerns/has_language.rb

```diff
@@ -0,0 +1,20 @@
+module HasLanguage
+
+  extend ActiveSupport::Concern
+
+  included do
+    validates :language, controlled_vocabulary_or_nil: { dictionary: 'LanguageDictionary' }
```

Comment:
You can pass a `allow_nil` option to the existing vocab validator to get this behaviour:
```
controlled_vocabulary: { dictionary: 'LanguageDictionary', allow_nil: true }
```